sycl/test/basic_tests/queue/queue_parallel_for_interface.cpp

```diff
@@ -75,4 +108,27 @@ int main() {
   test_range<class test_nd_range3, sycl::nd_range, 3>(q);
 
   test_number_braced_init_list(q);
+
+  auto e1 = q.parallel_for<class Event1>(1, [=](auto i) {
+    static_assert(std::is_same<decltype(i), sycl::item<1>>::value,
+                  "lambda arg type is unexpected");
+  });
+
+  auto e2 = q.parallel_for<class Event2>(1, [=](auto i) {
+    static_assert(std::is_same<decltype(i), sycl::item<1>>::value,
+                  "lambda arg type is unexpected");
+  });
```

Comment:
Is this identical to the test above it?  If this is deliberate, it might be a good idea to add a comment explaining why the second test is necessary.

Reply:
Done. Add the comment why it's necessary